modules/localgov_microsites_group_term_ui/config/install/views.view.lgms_group_taxonomy_terms.yml

```diff
@@ -449,8 +447,7 @@ display:
             type: 'entity:group'
             fail: 'not found'
           validate_options:
-            bundles:
```

Comment:
Should this be being removed? It seems correct that it's restricted to microsites.

Reply:
I was thinking it doesn't make any difference adding a group restriction and it makes the tests simpler (no dependency on the Microsite group), but happy to add it back.